code/game/machinery/camera/camera.dm

```diff
@@ -437,14 +436,15 @@ var/list/camera_messages = list()
 
 /obj/machinery/camera/Hear(var/datum/speech/speech, var/rendered_speech="")
 	if(isHearing())
+		var/datum/speech/copy = speech.clone()
+		copy = tv_message(copy)
```

Comment:
`tv_message` modifies the input speech datum, so you could avoid returning it and reassigning it